epyqlib/utils/qt.py

```diff
@@ -631,14 +631,14 @@ def dialog_from_file(parent, title, file_name):
     )
 
 
-class PySortFilterProxyModel(QtCore.QSortFilterProxyModel):
-    def __init__(self, *args, filter_column, **kwargs):
-        super().__init__(*args, **kwargs)
-
-        # TODO: replace with filterKeyColumn
-        self.filter_column = filter_column
+@attr.s
+class SortFilterProxyModel(QtCore.QSortFilterProxyModel):
+    parent = attr.ib(default=None)
+    filter_column = attr.ib(default=0)
+    wildcard = attr.ib(default=QtCore.QRegExp())
 
-        self.wildcard = QtCore.QRegExp()
+    def __attrs_post_init__(self):
+        super().__init__(self.parent)
         self.wildcard.setPatternSyntax(QtCore.QRegExp.Wildcard)
```

Comment:
I held off changing the `wildcard` declaration + initialization to a one liner. I did do the `_parent`. Again that's a good find. What I found is that no matter what, there is no way to avoid adding the `__attrs_post_init__` method. The inherited class __init__ method has to be called and as far as I can tell the only way to do that is making a call through `__attrs_post_init__`. 

Really, it doesn't make much sense to have the base class. It's not saving any lines of code, that's for sure. Let me know what you think.